sphinx/domains/python/__init__.py

```diff
@@ -824,6 +824,12 @@ def _make_module_refnode(self, builder: Builder, fromdocname: str, name: str,
         return make_refnode(builder, fromdocname, module.docname, module.node_id,
                             contnode, title)
 
+    def _intersphinx_adjust_object_types(self, objtypes: list[str]) -> None:
+        # we adjust the object types for backwards compatibility
+        if 'attribute' in objtypes:
```

Comment:
```suggestion
        if 'attribute' in objtypes and 'method' not in objtypes:
```

I wouldn't want to renact #11337 :smile:

Reply:
Indeed :-). I missed this when resurrecting the PR. Fixed.